internal/compiler/typeloader.rs

```diff
@@ -384,9 +404,15 @@ impl TypeLoader {
             }
             Err(err) if err.kind() == std::io::ErrorKind::NotFound => {
                 state.borrow_mut().diag.push_error(
-                        format!(
-                            "Cannot find requested import \"{file_to_import}\" in the include search path",
-                        ),
+                        if file_to_import.starts_with('@') {
+                            format!(
+                                "Cannot find requested library \"{file_to_import}\" in the library search path",
```

Comment:
Ah, but if the import is `@some-library/some_file.slint` the error message is going to call the entire string a library, when there are in fact two error cases:

1. `some-library` could not be found (missing in library search path)
2. Within `some-library` the file `some_file.slint` could not be found.